src/game/models/asteroid.rs

```diff
@@ -1,5 +1,16 @@
 //! Defines the asteroid component.
```

Comment:
In general, I prefer to have proper grammar in all of my comments/docs (Capital letters at the start of sentences, periods at the end, etc). But it's not a big deal and I can clean it up afterwards if you want.